### ISSUE TITLE
Does not work with VS2017 Open Folder/Cmake solutions 

### ISSUE THREAD

**Issue by ChrisAWills** [VioletGiraffe/cppcheck-vs-addin#187]
No results seem to be generated when trying to use the option to run after each save when opening the project using Visual Studio 2017's open folder option. 

Our project is made with CMAKE so no project file is made we just utilize the open folder.

Could it be made to work with this setup.

**Comment by mghstehr** [comment]
Yes (please do it), it would be very nice, if it could support CMAKE based "projects".

Best Regards
Matthias

**Comment by VioletGiraffe** [comment]
I agree, this would be a nice feature to have. But I don't use these kinds of projects, so I'm personally not motivated enough to work on this. If anyone wants to contribute, you are more than welcome!
